includes/blocks/class-sensei-block-helpers.php

```diff
@@ -73,10 +78,26 @@ public static function render_style_attributes( $class_names, $css ) {
 		$class_names = array_merge( is_array( $class_names ) ? $class_names : [ $class_names ], $css['css_classes'] );
 		return sprintf(
 			'class="%s" style="%s"',
-			esc_attr( implode( ' ', array_map( 'sanitize_html_class', $class_names ) ) ),
+			esc_attr( implode( ' ', $class_names ) ),
 			esc_attr( implode( '; ', $css['inline_styles'] ) )
 		);
 	}
 
+	/**
+	 * Add default style to list of classes if no style is selected.
+	 *
+	 * @param array $attributes Block attributes.
+	 *
+	 * @return string
+	 */
+	public static function block_class_with_default_style( $attributes ) {
+		$class_name = $attributes['className'] ?? '';
+		if ( empty( $class_name ) && false === strpos( $class_name, 'is-style-' ) ) {
+			$class_name .= ' is-style-default';
```

Comment:
The default style would not get added to the block class list, unless the user explicitly clicks it.